connect/mirror/README.md

```diff
@@ -194,6 +194,10 @@ it is important to keep configuration consistent across flows to the same
 target cluster. In most cases, your entire organization should use a single
 MM2 configuration file.
 
+### Exactly-once
```

Comment:
This will change when https://github.com/apache/kafka/pull/13137 is merged

Reply:
We merged https://github.com/apache/kafka/pull/13137 so this needs updating.

Reply:
Yes, sorry for the delay here. I realized that the integration test added in that PR does not actually enable exactly-once on MM2 due a configuration error.

I've corrected the error in this PR, updated the MM2 README to include instructions on enabling exactly-once support, and added `isolation.level = read_committed` to the integration test for good measure to ensure that the steps we recommend in the README can actually be followed successfully.